source/nodejs/adaptivecards/src/card-elements.ts

```diff
@@ -257,6 +291,44 @@ export abstract class CardElement {
         // Does nothing in base implementation
     }
 
+    handleOverflow(maxHeight: number) {
+        if (this.isVisible || this.isHiddenDueToOverflow()) {
+            var handled = this.truncateOverflows(maxHeight);
+
+            // Even if we were unable to truncate the element to fit this time,
+            // it still could have been previously truncated
+            this._truncatedDueToOverflow
+                = handled || this._truncatedDueToOverflow;
+
+            if (!handled) {
+                this.hideElementDueToOverflow();
+            }
+            else if (handled && !this._isVisible) {
+                this.showElementHiddenDueToOverflow();
+            }
+        }
+    }
+
+    resetOverflow(): boolean {
+        var sizeChanged = false;
+
+        if (this._truncatedDueToOverflow) {
+            this.undoOverflowTruncation();
+            this._truncatedDueToOverflow = false;
+            sizeChanged = true;
+        }
+
+        if (this.isHiddenDueToOverflow) {
+            this.showElementHiddenDueToOverflow();
+        }
+
+        return sizeChanged;
+    }
+
+    isOnPage(): boolean {
```

Comment:
Should that be renamed "overflows" or "isOverflowing"?

Reply:
This method actually doesn't check for overflow, it's just a check for whether the element has been added to the DOM yet.

Reply:
Ah got it. How about "isRendered" then?